.travis.yml

```diff
@@ -383,6 +386,23 @@ script:
     - oshrun -np 4 ./SHMEM/Synch_p2p/p2p 10 1000 1000
     - oshrun -np 4 ./SHMEM/Transpose/transpose 10 1000
     - make clean
+    ###
+    ### Tarball and RPM build check
+    ###
+    - >
+      if [ ! -z "$SOS_CHECK_TARBALL_RPM" ]; then
+          cd $SOS_SRC
+          ./configure --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple
+          make dist
+          rpmbuild -ta ./sos-*.tar.gz --define "configargs --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple"
+          tar zxvf sos-*.tar.gz
+          cd sos-*
+          ./autogen.sh
+          mkdir build
+          cd build
+          ../configure --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple
```

Comment:
`../configure --with-ofi=$TRAVIS_INSTALL/libfabric/ $SOS_GLOBAL_BUILD_OPTS $SOS_BUILD_OPTS`

Reply:
Oh. I thought those are the only two flags that we want to be tested and similar to what is expected during IFS release. Thanks, will update it.